Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1752,6 +2094,49 @@ private HandleType GetHandleType(Transform handle)
             return HandleType.None;
         }
 
+        private bool TryGetPointerPoint(Handedness handed, out Vector3 point)
+        {
+            IMixedRealityHand hand = HandJointUtils.FindHand(handed);
+            if (hand != null && hand.InputSource.Pointers[0].Result != null)
+            {
+                point = hand.InputSource.Pointers[0].Result.Details.Point;
+                hand = null;
```

Comment:
we don't have to set this here - we're returning anyways

Reply:
removed